src/io/flutter/inspector/DiagnosticsNode.java

```diff
@@ -38,6 +40,7 @@
   public DiagnosticsNode(JsonObject json, InspectorService inspectorService) {
     this.inspectorService = inspectorService;
     this.json = json;
+    this.widget = FlutterWidget.getCatalog().getWidget(getDescription());
```

Comment:
getDescription() isn't the right field to be using for this.
Add TODO(jacobr): provide the runtime type of the value of the node.

Reply:
Done.